<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>MarkerDialog_UI</class>
 <widget class="QDialog" name="MarkerDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="clip_filesize_2">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="TimecodeDisplay" name="m_in"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="range_marker_checkbox">
       <property name="text">
        <string>Range Marker</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_end_time">
       <property name="text">
        <string>End Time</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TimecodeDisplay" name="m_end">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_duration">
       <property name="text">
        <string>Duration</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="TimecodeDisplay" name="m_duration">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_category">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="clip_filesize_3">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="marker_comment"/>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="MarkerCategoryChooser" name="marker_category">
       <property name="allowAll">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="KCollapsibleGroupBox" name="multimarker_box">
     <property name="title">
      <string>Add multiple markers</string>
     </property>
     <property name="expanded">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="clip_filesize_5">
          <property name="text">
           <string>Count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="occurrences">
          <property name="minimum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="clip_filesize_4">
          <property name="text">
           <string>Interval</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TimecodeDisplay" name="interval"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="clip_thumb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Image preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimecodeDisplay</class>
   <extends>QWidget</extends>
   <header>widgets/timecodedisplay.h</header>
  </customwidget>
  <customwidget>
   <class>MarkerCategoryChooser</class>
   <extends>QComboBox</extends>
   <header>widgets/markercategorychooser.h</header>
  </customwidget>
  <customwidget>
   <class>KCollapsibleGroupBox</class>
   <extends>QWidget</extends>
   <header>kcollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MarkerDialog_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MarkerDialog_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>range_marker_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_end_time</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>range_marker_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_end</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>range_marker_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_duration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>range_marker_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_duration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
